tsconfig.json

```diff
@@ -15,6 +15,6 @@
     "isolatedModules": true,
     "jsx": "preserve"
   },
-  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx"],
+  "include": ["next-env.d.ts", "markdoc_nextjs_tags.d.ts", "**/*.ts", "**/*.tsx"],
```

Comment:
Is we move that global definition into https://github.com/markdoc/next.js/blob/main/src/index.js, then we don't need to include this change.